docs/coordinates/satellites.rst

```diff
@@ -0,0 +1,84 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-satellites:
+
+Working with Earth satellites using Astropy Coordinates
+*******************************************************
+
+Finding ``TEME`` coordinates from Two-Line-Ephemerides (TLE)
+============================================================
+
+There is currently no support in `astropy.coordinates` for computing satellite orbits
+from the TLE orbital elements avaiable from a source like `Celestrak <http://celestrak.com/>`_.
+Full support for handling TLE files is available in the `Skyfield <http://rhodesmill.org/skyfield/>`_
+library, but if you do not wish to install that library, advice for dealing with satellite data in
+astropy is below.
+
+..
+  EXAMPLE START
+  Using spg4 to get a TEME coordinate
+
+You will need some external library to compute the position and velocity of the satellite from the
+TLE orbital elements. The `spgp4 <https://pypi.org/project/sgp4/>`_ library can do this. An example
+using this library to find the  `~astropy.coordinates.TEME` coordinates of a satellite is::
+
+        >>> from sgp4.api import Satrec
+        >>> from sgp4.api import SGP4_ERRORS
+        >>> s = '1 25544U 98067A   19343.69339541  .00001764  00000-0  38792-4 0  9991'
+        >>> t = '2 25544  51.6439 211.2001 0007417  17.6667  85.6398 15.50103472202482'
+        >>> satellite = Satrec.twoline2rv(s, t)
+
+The ``satellite`` object has a method, ``satellite.sgp4``, that will try to compute the TEME position
+and velocity at a given time::
+
+        >>> from astropy.time import Time
+        >>> t = Time(2458827.362605, format='jd')
+        >>> error_code, teme_p, teme_v = satellite.sgp4(t.jd1, t.jd2)  # in km and km/s
+        >>> if error_code != 0:
```

Comment:
Can this happen given this fixed position/time example?

Reply:
Not really, but I'd like to leave it in to illustrate best practice.